src/app/components/dropdown/index.html

```diff
@@ -1,15 +1,176 @@
+        </div>
+    </sky-demo-page-content>
+    
+    <sky-demo-page-content sectionHeading="Content">
+      <h3>Dropdown menu action text</h3>
+      <sky-docs-visual-example>
+        <sky-docs-visual-example-text>
+          <p>If an action clearly applies to the object associated with the context menu, use the following format for the action wording:</p>
+          <p><span class="sky-emphasized">&lt;Verb&gt;</span></p>
+        </sky-docs-visual-example-text>
+        <sky-docs-visual-example-image>
+          <img class="bordered" src="~/assets/img/guidelines/wording/context-menu.png" alt="context menu actions: View, Edit, Delete" />
+        </sky-docs-visual-example-image>
+      </sky-docs-visual-example>
+      <sky-docs-visual-example>
+        <sky-docs-visual-example-text>
+          <p>For a dropdown button with one type of action, use the following format:</p>
+          <p>Button text: <span class="sky-emphasized">&lt;Verb&gt;</span></p>
+          <p>Menu item: <span class="sky-emphasized">&lt;Direct object&gt;</span></p>
+        </sky-docs-visual-example-text>
+        <sky-docs-visual-example-image>
+          <img class="bordered" src="~/assets/img/guidelines/wording/dropdown-single-action.png" alt="dropdown menu Edit: Addresses, Email addresses, Phone numbers" />
+        </sky-docs-visual-example-image>
+      </sky-docs-visual-example>
+      <sky-docs-visual-example>
+        <sky-docs-visual-example-text>
+          <p>For a dropdown button with more than one type of action, use the following format:</p>
+          <p>Button text: <span class="sky-emphasized">More</span></p>
+          <p>Button icon: <span class="sky-emphasized">fa-ellipsis-h</span></p>
+          <p>Menu item: <span class="sky-emphasized">&lt;Verb&gt; &lt;direct object&gt;</span></p>
+        </sky-docs-visual-example-text>
+        <sky-docs-visual-example-image>
+          <img class="bordered" src="~/assets/img/guidelines/wording/more-button.png" alt="dropdown menu More: Place vendor on hold, Archive vendor" />
+        </sky-docs-visual-example-image>
+      </sky-docs-visual-example>
+      
+        <div class="sky-text-block sky-margin-stacked-content-section">
+        <p>Additional button text guidelines:</p>
+				<ul>
+					<li>Use sentence-style capitalization.</li>
+					<li>The direct object can be singular or plural depending on whether the action acts on one or multiple items.</li>
+					<li>Don’t append extra words such as “details.” For example, use “Edit constituent” rather than “Edit constituent details.”</li>
+				</ul>
+      </div>
+    </sky-demo-page-content>
+      
+    <sky-demo-page-content sectionHeading="Layout">
+        
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>Place context menus to the left of all row-specific content, but to the right of all list controls (such as checkboxes in a grid or repeater, or the expand-collapse actions in a tree view).</p>
```

Comment:
It's minor, but I'd use a such as clause instead of parentheses. I'm going to make that change, but we can undo it if anyone objects. I'll also change "context menus" to "context menu dropdowns" because I think that's more accurate. I'll also delete the comma and use plural throughout.